Default URIs do not support windows file paths.

I am seeing issue #22 even with defaults when building on Windows.

Here is the contents of my pom.xml

```
<?xml version="1.0" encoding="UTF-8"?>
<project xmlns="http://maven.apache.org/POM/4.0.0"
         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
    <parent>
        <artifactId>backend</artifactId>
        <groupId>com.flavorate</groupId>
        <version>0.1.0</version>
    </parent>
    <modelVersion>4.0.0</modelVersion>

    <artifactId>deploy-charts</artifactId>

    <build>
        <plugins>
            <plugin>
                <groupId>org.microbean</groupId>
                <artifactId>helm-maven-plugin</artifactId>
                <version>2.8.2.1.1.1</version>
                <executions>
                    <execution>
                        <id>Package your-project chart</id>
                        <phase>generate-sources</phase> <!-- pick a suitable phase, though this seems good -->
                        <goals>
                            <goal>package</goal>
                        </goals>
                    </execution>
                </executions>
            </plugin>
        </plugins>
    </build>

</project>
```

Here is the output of mvn package

```
[INFO] Scanning for projects...
[INFO] ------------------------------------------------------------------------
[INFO] Detecting the operating system and CPU architecture
[INFO] ------------------------------------------------------------------------
[INFO] os.detected.name: windows
[INFO] os.detected.arch: x86_64
[INFO] os.detected.version: 10.0
[INFO] os.detected.version.major: 10
[INFO] os.detected.version.minor: 0
[INFO] os.detected.classifier: windows-x86_64
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building deploy-charts 0.1.0
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- helm-maven-plugin:2.8.2.1.1.1:package (Package your-project chart) @ deploy-charts ---
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 5.055 s
[INFO] Finished at: 2018-07-23T20:42:48-05:00
[INFO] Final Memory: 20M/222M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.microbean:helm-maven-plugin:2.8.2.1.1.1:package (Package your-project chart) on project deploy-charts: Unable to parse configuration of mojo org.microbean:helm-maven-plugin:2.8.2.1.1.1:package for parameter chartContentsUri: Cannot convert 'file:C:\Users\kalex\Sanity\flavorate\flavorate-backend\deploy-charts/src/helm/charts/deploy-charts/' to URI: Illegal character in opaque part at index 7: file:C:\Users\kalex\Sanity\flavorate\flavorate-backend\deploy-charts/src/helm/charts/deploy-charts/ -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginConfigurationException
```

--- a/src/main/java/org/microbean/helm/maven/PackageMojo.java
+++ b/src/main/java/org/microbean/helm/maven/PackageMojo.java
@@ -23,6 +23,7 @@ import java.io.OutputStream;
 
 import java.net.UnknownServiceException;
 import java.net.URI;
+import java.net.URISyntaxException;
 import java.net.URL;
 import java.net.URLConnection;
 
@@ -91,10 +92,11 @@ public class PackageMojo extends AbstractHelmMojo implements Disposable {
   private boolean skip;
 
   /**
-   * A {@link URI} pointing to the Helm chart contents to load.
+   * A {@link String} in {@link URI} form pointing to the Helm chart
+   * contents to load.
    */
   @Parameter(required = true, defaultValue = "file:${project.basedir}/src/helm/charts/${project.artifactId}/")
-  private URI chartContentsUri;
+  private String chartContentsUri;
   
   @Parameter
   private AbstractChartLoader<URL> chartLoader;
@@ -369,11 +371,32 @@ public class PackageMojo extends AbstractHelmMojo implements Disposable {
   }
 
   public URI getChartContentsUri() {
-    return this.chartContentsUri;
+    final URI returnValue;
+    if (this.chartContentsUri == null) {
+      returnValue = null;
+    } else {
+      URI temp = null;
+      try {
+        temp = new URI(this.chartContentsUri);
+      } catch (final URISyntaxException uriSyntaxException) {
+        try {
+          temp = new URI(this.chartContentsUri.replace('\\', '/'));
+        } catch (final URISyntaxException backslashReplacementFailed) {
+          throw new IllegalStateException(backslashReplacementFailed.getMessage(), backslashReplacementFailed);
+        }
+      } finally {
+        returnValue = temp;
+      }
+    }
+    return returnValue;
   }
 
   public void setChartContentsUri(final URI chartContentsUri) {
-    this.chartContentsUri = chartContentsUri;
+    if (chartContentsUri == null) {
+      this.chartContentsUri = null;
+    } else {
+      this.chartContentsUri = chartContentsUri.toString();
+    }
   }
 
   public URI getChartTargetUri() {

